src/Message/AbstractRequest.php

```diff
@@ -204,6 +277,49 @@ public function setSource($value)
         return $this->setParameter('source', $value);
     }
 
+    /**
+     * @param       $data
+     * @param array $headers
+     *
+     * @return \Guzzle\Http\Message\RequestInterface
+     */
+    protected function createClientRequest($data, array $headers = null)
+    {
+        // Stripe only accepts TLS >= v1.2, so make sure Curl is told
+        $config                          = $this->httpClient->getConfig();
+        $curlOptions                     = $config->get('curl.options');
+        $curlOptions[CURLOPT_SSLVERSION] = 6;
+        $config->set('curl.options', $curlOptions);
+        $this->httpClient->setConfig($config);
+
+        // Use custom query aggregator because Stripe only supports a specific format
```

Comment:
Explain _why_. It can be simply a reference to the documentation for `StripeQueryAggregator`